sa/database.go

```diff
@@ -61,6 +61,8 @@ func NewDbMapFromConfig(config *mysql.Config, maxOpenConns int) (*gorp.DbMap, er
 		return nil, err
 	}
 	setMaxOpenConns(db, maxOpenConns)
+	// Set a nonzero idle connection limit so we get better connection reuse.
+	db.SetMaxIdleConns(20)
```

Comment:
I think its probably worth the time to make this a config parameter. If it turns out that using `20` causes its own performance problems it would be a shame to have to revert the deploy & wait another full cycle to try a new value.

Reply:
Yep, you're totally right. I was thinking "I'm highly confident this is safe, since it just governs idle connections, and we generally have around 20 active connections anyhow." But of course it's usually the changes we think are "safe" that cause unexpected issues. I'll add the config.